File: woodroid/src/main/java/me/gilo/woodroid/models/filters/RefundFilter.java
package me.gilo.woodroid.models.filters;

public class RefundFilter extends ListFilter{

    private int[] parent;
    private int[] parent_exclude;

    int dp;

    public int[] getParent() {
        return parent;
    }

    public void setParent(int[] parent) {
        this.parent = parent;
        addFilter("parent", parent);
    }

    public int[] getParent_exclude() {
        return parent_exclude;
    }

    public void setParent_exclude(int[] parent_exclude) {
        this.parent_exclude = parent_exclude;

        addFilter("parent_exclude", parent_exclude);
    }

    public int getDp() {
        return dp;
    }

    public void setDp(int dp) {
        this.dp = dp;
        addFilter("dp", dp);

    }
}

File: woodroid/src/main/java/me/gilo/woodroid/models/filters/RefundFilter.kt
package me.gilo.woodroid.models.filters

class RefundFilter : ListFilter() {

    var parent: IntArray? = null
        set(parent) {
            field = parent
            if (parent != null) {
                addFilter("parent", parent)
            }
        }
    var parent_exclude: IntArray? = null
        set(parent_exclude) {
            field = parent_exclude

            if (parent_exclude != null) {
                addFilter("parent_exclude", parent_exclude)
            }
        }

    internal var dp: Int = 0

    fun getDp(): Int {
        return dp
    }

    fun setDp(dp: Int) {
        this.dp = dp
        addFilter("dp", dp)

    }
}
